plugins/inventory/now.py

```diff
@@ -110,6 +110,11 @@
       - Host variable to use as I(ansible_host) when generating inventory hosts.
     type: str
     default: ip_address
+    deprecated:
+      why: Constructed features made this obsolete
+      version: 2.0.0
```

Comment:
Wouldn't this be 1.2.0 since it's technically already deprecated in the upcoming release, or does this imply the version when this feature gets removed from the module?
```suggestion
      version: 1.2.0
```

Reply:
Those versions inform users in what version deprecated stuff will be removed. So 2.0.0 is OK here.